src/ngeo.js

```diff
@@ -13,3 +13,33 @@ ngeo.module = angular.module('ngeo', []);
  * @type {string}
  */
 ngeo.baseTemplateUrl = 'ngeo';
+
+
+/**
+ * @enum {string}
+ */
+ngeo.FeatureProperties = {
```

Comment:
I'll fix this.  Sorry, I'm used to OL3 doing it this way, see: https://github.com/openlayers/ol3/blob/master/src/ol/layer/layerbase.js#L12